catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -61,13 +62,17 @@
 import ddf.catalog.federation.FederationException;
 import ddf.catalog.filter.FilterBuilder;
 import ddf.catalog.impl.filter.GeoToolsFunctionFactory;
+import ddf.catalog.operation.QueryRequest;
 import ddf.catalog.operation.QueryResponse;
 import ddf.catalog.operation.impl.QueryImpl;
 import ddf.catalog.operation.impl.QueryRequestImpl;
 import ddf.catalog.source.SourceUnavailableException;
 import ddf.catalog.source.UnsupportedQueryException;
 
 public class EndpointUtil {
+
+    public static final Integer QUERY_DEFAULT_PAGE_SIZE = 2000000000;
```

Comment:
one nit: could use number separators `2_000_000_000`
Second, Is there any way we can use the constant that is already defined for the solr cloud stuff? I see that they have an undocumented system property which I'm sure would be allowable to use. The property name is private so we would have to use the string literal.
`Integer.valueOf(System.getProperty("solr.query.rows",  Integer.toString(2_000_000_000)))`